lib/suppressions.cpp

```diff
@@ -302,22 +304,28 @@ bool Suppressions::Suppression::parseComment(std::string comment, std::string *e
     if (comment.compare(comment.size() - 2, 2, "*/") == 0)
         comment.erase(comment.size() - 2, 2);
 
+    const std::string cppchecksuppress = "cppcheck-suppress";
+
     std::istringstream iss(comment.substr(2));
     std::string word;
     iss >> word;
-    if (word != "cppcheck-suppress")
+    if (word.substr(0, cppchecksuppress.size()) != cppchecksuppress)
```

Comment:
I believe we should also return false if word would be "cppcheck-suppressabcdef"

Reply:
you could use the startsWith that you added.